tools/flowc/lsp_types.flow

```diff
@@ -0,0 +1,189 @@
+import formats/json/json;
+import formats/uri2/uri2;
+
+export {
```

Comment:
Add a comment with the LSP link and description.

Reply:
Added.